src/MeetingsSDKAdapter.test.js

```diff
@@ -179,47 +173,37 @@ describe('Meetings SDK Adapter', () => {
       meetingSDKAdapter.getStream(meetingID, {sendAudio: true})
         .pipe(last())
         .subscribe((localMedia) => {
-          expect(localMedia).toMatchObject({
-            stream: mockSDKMediaStreams.localAudio,
-            permission: 'ALLOWED',
-          });
+          expect(localMedia.stream).toMatchMediaStream(mockSDKMediaStreams.localAudio);
+
+          expect(localMedia.permission).toBe('ALLOWED');
+
           done();
         });
       meetingSDKAdapter.getStream(meetingID, {sendVideo: true}).pipe(
         last(),
       ).subscribe((localMedia) => {
-        expect(localMedia).toMatchObject({
-          stream: mockSDKMediaStreams.localVideo,
-          permission: 'ALLOWED',
-        });
+        expect(localMedia.stream).toMatchMediaStream(mockSDKMediaStreams.localVideo);
+
+        expect(localMedia.permission).toBe('ALLOWED');
+
         done();
       });
     });
   });
 
   describe('attachMedia()', () => {
     let event;
-    let mockMediaStreamInstance;
 
     beforeEach(() => {
-      mockMediaStreamInstance = {
-        getAudioTrack: () => [],
-        getVideoTrack: () => [],
-      };
       event = {
         type: 'local',
-        stream: {
-          getAudioTracks: jest.fn(() => mockSDKMediaStreams.localAudio),
-          getVideoTracks: jest.fn(() => mockSDKMediaStreams.localVideo),
-        },
+        stream: mockSDKMediaStreams.localAudioVideo,
       };
-
-      global.MediaStream = jest.fn(() => mockMediaStreamInstance);
     });
 
     test('keeps `localAudio.stream` empty, if the event type is `local` and the audio stream is empty', () => {
       meetingSDKAdapter.meetings[meetingID] = {
-        disabledLocalAudio: {},
+        disabledLocalAudio: mockSDKMediaStreams.localAudio,
```

Comment:
this is a (very) incomplete meeting object, it would be more robust to have a complete object here:
```js
meetingSDKAdapter.meetings[meetingID].disabledLocalAudio = ...;
mmeetingSDKAdapter.meetings[meetingID].localAudio = {stream: null, permission: 'ALLOWED'};
```

Reply:
or you can use the same approach in line 352 https://github.com/webex/sdk-component-adapter/pull/170/files#diff-94a8fe68ea1a2cb38f44e0058884a91f48b7e685cbd6d6b1c88152c5d4024627R542